src/vcSettingsUI.cpp

```diff
@@ -940,6 +940,23 @@ void vcSettingsUI_BasicMapSettings(vcState *pProgramState)
     vcSettings_ApplyMapChange(&pProgramState->settings);
     vcRender_ClearTiles(pProgramState->pRenderContext);
   }
+
+  if (ImGui::Button(vcString::Get("settingsMapECEFMode")))
+  {
+    static const int32_t s_ECEFModeID = 4978;
+    static int32_t s_previousSRID = -1;
```

Comment:
This variable should live in `vcState` or `vcProject` rather than here, that way a project change can update it as well so that it isn't carried through logout or changing projects.